test/automation/src/sql/configurePythonDialog.ts

```diff
@@ -39,7 +39,7 @@ export class ConfigurePythonDialog extends Dialog {
 
 	private async _waitForInstallationComplete(): Promise<void> {
 		const installationCompleteNotification = '.notifications-toasts div[aria-label="Notebook dependencies installation is complete, source: Notebook Core Extensions (Extension), notification"]';
-		await this.code.waitForElement(installationCompleteNotification);
+		await this.code.waitForElement(installationCompleteNotification, undefined, 600); // wait up to 1 minute for python installation
```

Comment:
is this 1 minute or 10 minutes?

Reply:
1 minute, the retry interval is 100 milliseconds (https://github.com/microsoft/azuredatastudio/blob/d2bdd2bace223ecc9436eee27b47b2618a353de6/test/automation/src/code.ts#L202)

Reply:
I see. Can we overwrite the retry interval for this particular test? Not sure that we need to check every 100 ms if we know that it takes > 20 seconds in some cases.

Reply:
Hmm there isn't a way to change the retry interval right now. We can add a parameter to waitForElement() but that would be a vscode change.

Reply:
gotcha, ok!